source/resume.html

```diff
@@ -0,0 +1,189 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="x-ua-compatible" content="ie=edge">
+
+  <title>Pot Pie Digital - Resume</title>
+  <meta name="description" content="Pot Pie Digital is a collaboration of Ashley Siebels, Mark Hayes, and Matt Hayes - created March, 2016">
+  <meta name="author" content="Ashley Siebels, Mark Hayes, and Matt Hayes">
+  <link rel="author" href="./humans.txt" />
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <!--
+  <link rel="apple-touch-icon" href="./apple-touch-icon.png">
+  <link rel="shortcut icon" href="./favicon.ico">
+  -->
+
+  <link rel="stylesheet" href="./stylesheet.css">
+  <link href="https://fonts.googleapis.com/css?family=Alegreya|Source+Sans+Pro" rel="stylesheet">
+</head>
+
+<body>
+  <main>
+    <header>
+      <nav>
+        <ul>
+          <li>
+            <a href="index.html#about">About</a>
```

Comment:
I _think_ these could all just be `/#whatever` since when it's deployed it'll be living at `https://potpiedigital.github.io/`.